app/newRelicSnippet.tsx

```diff
@@ -0,0 +1,16 @@
+;NREUM.loader_config={accountID:"3340026",trustKey:"3340026",agentID:"1385887337",licenseKey:"NRJS-94c48cdf4bf4fa5718d",applicationID:"1385887337"}
+;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"NRJS-94c48cdf4bf4fa5718d",applicationID:"1385887337",sa:1}
+`;
+
+export const NewRelicSnippet = () => {
+  return (
+    <script
```

Comment:
Thoughts on using next/Script here? https://nextjs.org/docs/basic-features/script

Reply:
So I tried by just converting this to Script. (I might be doing it wrong)
I can't seem to get it to load on the page though. 

I tried even just adding this 

![image](https://user-images.githubusercontent.com/5621207/141725572-b264197c-ed51-4716-8c1e-5bfab0748aa2.png)


Added that to the _document, _app, index.tsx. 

Nothing. What am I doing wrong?

Reply:
nvm figured it out, I was testing it incorrectly 😓